web/src/narrow_banner.js

```diff
@@ -148,6 +149,14 @@ function pick_empty_narrow_banner() {
             return SPECTATOR_STREAM_NARROW_BANNER;
         }
 
+        // If at least one of the operators in the empty narrow is the "is" operator,
+        // we only display the title.
+        if (current_filter.operands("is").length > 0) {
+            return {
+                title: default_title,
```

Comment:
Should we be using `empty_search_narrow_title`, i.e. the "No search results" case, here?

I think there's a question of whether we need to filter on `is:` here, or whether we should just change the `return default_banner` just below into returning `empty_search_narrow_title`, rather than adding this new conditional.

Reply:
"No search results" might be better... I don't fully understand the set of views covered by this PR, but the way you get to them is via the search bar, right?

Reply:
Yeah, you get to them via the search bar.

More specifically, they would:
- have more than one search filter
- one of the filters would be "is:..."
- there would be no key word or phrase in the search

Some examples:
- `sender:me is:private`
- `stream:Rome is:resolved is:unread`
- `pm-with:cordelia is:starred`

I think in the [relevant CZO conversation](https://chat.zulip.org/#narrow/stream/9-issues/topic/empty.20feed.20message/near/1511452), we landed on removing the second sentence.

But it would be completely reasonable to have the 'No search results." case for all empty searches with multiple filters. I think that the moment we reserve that for searches with key words and phrases. Maybe circle back on CZO?

Reply:
Correct, via the search bar or someone sending you a link copy-pasted from the search bar.

Reply:
I have dropped a message on CZO for a better discussion over this.